PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -109,31 +189,21 @@ struct StoreAppConditionalView: View {
             }
         }
         .task {
-            iconUrl = await getIconURLFromBundleIdentifier(app.id, app.region)
+            itunesData = await getITunesData(app.itunesLookup)
```

Comment:
Leaving the "itunesLookup" attribute empty `""` for apps not available in the App Store for instance, results in a crash. This isn't desired behavior, as PlayCover should let you list apps that are not available in the App Store.